Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -242,18 +243,29 @@ private bool IsEligibleForMultiVersion(string folderName, string testFilename)
             return false;
         }
 
-        private List<VideoFileInfo> GetExtras(IEnumerable<VideoFileInfo> remainingFiles, List<string> baseNames)
+        private IEnumerable<VideoFileInfo> GetExtras(IEnumerable<VideoFileInfo> remainingFiles, List<string> baseNames)
         {
             foreach (var name in baseNames.ToList())
             {
                 var trimmedName = name.TrimEnd().TrimEnd(_options.VideoFlagDelimiters).TrimEnd();
                 baseNames.Add(trimmedName);
             }
 
-            return remainingFiles
```

Comment:
why change this?

Reply:
Again a long and complicated linq expression.